extensions/GlobalMessages/classes/GlobalMessagesService.class.php

```diff
@@ -7,7 +7,8 @@
 class GlobalMessagesService {
 	const I18N_FILE_EXTENSION = ".i18n.php";
 	const PHP_FILE_EXTENSION = ".php";
-	const CORE_LOCALISATION_FILE_REGEX = "/^.+GlobalMessages[A-Z]+\.i18n\.php$/i";
+	// const CORE_LOCALISATION_FILE_REGEX = "/^.+GlobalMessages[A-Z]+\.i18n\.php$/i";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS125)